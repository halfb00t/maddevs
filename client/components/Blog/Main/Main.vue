<template>
  <main class="home">
    <BlogHeader />
    <LatestPostsSection />
    <NewCustomerUniversitySection />
    <AllPostsSection :all-posts="allPosts" />
  </main>
</template>

<script>
import { mapActions, mapGetters } from 'vuex'
import BlogHeader from '@/components/Blog/Main/BlogHeader'
import LatestPostsSection from '@/components/Blog/Main/LatestPostsSection'
import NewCustomerUniversitySection from '@/components/Blog/Main/NewCustomerUniversitySection'
import AllPostsSection from '@/components/Blog/Main/AllPostsSection'

export default {
  name: 'Main',
  components: {
    BlogHeader,
    LatestPostsSection,
    NewCustomerUniversitySection,
    AllPostsSection,
  },

  data() {
    return {
      pageSize: 12,
    }
  },

  computed: {
    ...mapGetters([
      'allPosts',
    ]),
  },

  created() {
    this.getContent()
  },

  methods: {
    ...mapActions(['getBlogPosts', 'getBlogAuthors', 'getCustomerUniversityContent']),

    getContent() {
      // Query to get blog home content
      this.getCustomerUniversityContent()

      // Query to get posts content to preview
      this.getBlogPosts()
      this.getBlogAuthors()
    },
  },
}
</script>
