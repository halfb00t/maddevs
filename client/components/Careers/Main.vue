<template>
  <div class="careers">
    <Banner />
    <LazyHydrate when-visible>
      <ProjectsCountries />
    </LazyHydrate>
    <LazyHydrate when-visible>
      <Benefits />
    </LazyHydrate>
    <LazyHydrate when-visible>
      <KeyMetrics />
    </LazyHydrate>
    <div class="careers_white-section">
      <LazyHydrate when-visible>
        <EmployeesBenefits />
      </LazyHydrate>
      <LazyHydrate when-visible>
        <OpenPositions />
      </LazyHydrate>
      <LazyHydrate when-visible>
        <CTABanner />
      </LazyHydrate>
    </div>
    <ScrollToPositionsLink />
  </div>
</template>

<script>
import LazyHydrate from 'vue-lazy-hydration'
import { mapActions } from 'vuex'
import Banner from '@/components/Careers/Banner'
import ScrollToPositionsLink from '@/components/Careers/shared/ScrollToPositionsLink'

export default {
  name: 'Main',
  components: {
    LazyHydrate,
    Banner,
    ProjectsCountries: () => import('@/components/Careers/ProjectsCountries'),
    Benefits: () => import('@/components/Careers/Benefits'),
    KeyMetrics: () => import('@/components/Careers/KeyMetrics'),
    EmployeesBenefits: () => import('@/components/Careers/EmployeesBenefits'),
    OpenPositions: () => import('@/components/Careers/OpenPositions'),
    CTABanner: () => import('@/components/Careers/CTABanner'),
    ScrollToPositionsLink,
  },

  mounted() {
    // Query to get vacancies and categories
    this.getCareersHome()
  },

  methods: {
    ...mapActions(['getCareersHome']),
  },
}
</script>

<style lang="scss" scoped>
.careers {
  &_white-section {
    background-color: $bgcolor--white-primary;
  }
}
</style>
