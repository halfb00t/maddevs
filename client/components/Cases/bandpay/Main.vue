<template>
  <section>
    <LazyHydrate when-visible>
      <About />
    </LazyHydrate>
    <LazyHydrate when-visible>
      <OpportunitiesCards />
    </LazyHydrate>
    <LazyHydrate when-visible>
      <BenefitMostBandPay />
    </LazyHydrate>
    <LazyHydrate when-visible>
      <ChallengesSolutions />
    </LazyHydrate>
    <LazyHydrate when-visible>
      <LookingAtTheFrontend />
    </LazyHydrate>
    <LazyHydrate when-visible>
      <IntegratingOnline />
    </LazyHydrate>
    <LazyHydrate when-visible>
      <TwoSmartphones />
    </LazyHydrate>
    <LazyHydrate when-visible>
      <WhatIsBackend />
    </LazyHydrate>
    <LazyHydrate when-visible>
      <BandPayAppArchitecture />
    </LazyHydrate>
    <LazyHydrate when-visible>
      <ImplementingPipelines />
    </LazyHydrate>
    <LazyHydrate when-visible>
      <BuyerRegistration />
    </LazyHydrate>
    <LazyHydrate when-visible>
      <ProjectDevelopment />
    </LazyHydrate>
    <LazyHydrate when-visible>
      <ProjectPlanning />
    </LazyHydrate>
    <LazyHydrate when-visible>
      <KeyAchievements />
    </LazyHydrate>
  </section>
</template>
<script>
import LazyHydrate from 'vue-lazy-hydration'

export default {
  name: 'Main',
  components: {
    LazyHydrate,
    About: () => import('@/components/Cases/bandpay/About'),
    OpportunitiesCards: () => import('@/components/Cases/bandpay/OpportunitiesCards'),
    BenefitMostBandPay: () => import('@/components/Cases/bandpay/BenefitMostBandPay'),
    ChallengesSolutions: () => import('@/components/Cases/bandpay/ChallengesSolutions'),
    LookingAtTheFrontend: () => import('@/components/Cases/bandpay/LookingAtTheFrontend'),
    IntegratingOnline: () => import('@/components/Cases/bandpay/IntegratingOnline'),
    TwoSmartphones: () => import('@/components/Cases/bandpay/TwoSmartphones'),
    WhatIsBackend: () => import('@/components/Cases/bandpay/WhatIsBackend'),
    BandPayAppArchitecture: () => import('@/components/Cases/bandpay/BandPayAppArchitecture'),
    ImplementingPipelines: () => import('@/components/Cases/bandpay/ImplementingPipelines'),
    BuyerRegistration: () => import('@/components/Cases/bandpay/BuyerRegistration'),
    ProjectDevelopment: () => import('@/components/Cases/bandpay/ProjectDevelopment'),
    ProjectPlanning: () => import('@/components/Cases/bandpay/ProjectPlanning'),
    KeyAchievements: () => import('@/components/Cases/bandpay/KeyAchievements'),
  },

}
</script>
