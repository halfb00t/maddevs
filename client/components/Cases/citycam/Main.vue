<template>
  <section>
    <LazyHydrate when-visible>
      <About />
    </LazyHydrate>
    <LazyHydrate when-visible>
      <WhyCityCam />
    </LazyHydrate>
    <LazyHydrate when-visible>
      <ChallengesAndSolutions />
    </LazyHydrate>
    <LazyHydrate when-visible>
      <Prototype />
    </LazyHydrate>
    <LazyHydrate when-visible>
      <PowerSupply />
    </LazyHydrate>
    <LazyHydrate when-visible>
      <DevicePerfecting />
    </LazyHydrate>
    <LazyHydrate when-visible>
      <OngoingProgress />
    </LazyHydrate>
    <LazyHydrate when-visible>
      <Technologies />
    </LazyHydrate>
    <LazyHydrate when-visible>
      <Team />
    </LazyHydrate>
  </section>
</template>

<script>
import LazyHydrate from 'vue-lazy-hydration'

export default {
  name: 'Main',
  components: {
    LazyHydrate,
    About: () => import('@/components/Cases/citycam/About'),
    WhyCityCam: () => import('@/components/Cases/citycam/WhyCityCam'),
    ChallengesAndSolutions: () => import('@/components/Cases/citycam/ChallengesAndSolutions'),
    Prototype: () => import('@/components/Cases/citycam/Prototype'),
    PowerSupply: () => import('@/components/Cases/citycam/PowerSupply'),
    DevicePerfecting: () => import('@/components/Cases/citycam/DevicePerfecting'),
    OngoingProgress: () => import('@/components/Cases/citycam/OngoingProgress'),
    Technologies: () => import('@/components/Cases/citycam/Technologies'),
    Team: () => import('@/components/Cases/citycam/Team'),
  },
}
</script>
