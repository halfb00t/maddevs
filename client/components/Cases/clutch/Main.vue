<template>
  <section>
    <LazyHydrate :when-visible="{rootMargin: '100px'}">
      <About />
    </LazyHydrate>
    <LazyHydrate :when-visible="{rootMargin: '100px'}">
      <WhyClutch />
    </LazyHydrate>
    <LazyHydrate :when-visible="{rootMargin: '100px'}">
      <TechChallenges />
    </LazyHydrate>
    <LazyHydrate when-visible>
      <OutStuffing />
    </LazyHydrate>
    <LazyHydrate when-visible>
      <Microservices />
    </LazyHydrate>
    <LazyHydrate when-visible>
      <DataAndPasswords />
    </LazyHydrate>
    <LazyHydrate when-visible>
      <WidgetSpeed />
    </LazyHydrate>
    <LazyHydrate when-visible>
      <DataProcessed />
    </LazyHydrate>
    <LazyHydrate when-visible>
      <DevOps />
    </LazyHydrate>
    <LazyHydrate when-visible>
      <Workflow />
    </LazyHydrate>
    <LazyHydrate when-visible>
      <Pipelines />
    </LazyHydrate>
    <LazyHydrate when-visible>
      <Standup />
    </LazyHydrate>
    <LazyHydrate when-visible>
      <TechStack />
    </LazyHydrate>
    <LazyHydrate when-visible>
      <Review />
    </LazyHydrate>
  </section>
</template>

<script>
import LazyHydrate from 'vue-lazy-hydration'

export default {
  name: 'Main',
  components: {
    Review: () => import('@/components/Cases/clutch/Review'),
    TechStack: () => import('@/components/Cases/clutch/TechStack'),
    Standup: () => import('@/components/Cases/clutch/Standup'),
    Pipelines: () => import('@/components/Cases/clutch/Pipelines'),
    Workflow: () => import('@/components/Cases/clutch/Workflow'),
    DevOps: () => import('@/components/Cases/clutch/DevOps'),
    DataProcessed: () => import('@/components/Cases/clutch/DataProcessed'),
    WidgetSpeed: () => import('@/components/Cases/clutch/WidgetSpeed'),
    DataAndPasswords: () => import('@/components/Cases/clutch/DataAndPasswords'),
    TechChallenges: () => import('@/components/Cases/clutch/TechChallenges'),
    Microservices: () => import('@/components/Cases/clutch/Microservices'),
    WhyClutch: () => import('@/components/Cases/clutch/WhyClutch'),
    OutStuffing: () => import('@/components/Cases/clutch/OutStuffing'),
    About: () => import('@/components/Cases/clutch/About'),
    LazyHydrate,
  },
}
</script>
