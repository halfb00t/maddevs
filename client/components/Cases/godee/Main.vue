<template>
  <main class="case">
    <CaseHeader
      :logo="logo"
      project="godee"
      video-name="/videos/godee-case-main-video.mp4"
      video-fallback-path="/images/Cases/godee/jpg/godee-case-preview.jpg"
    >
      <h1
        slot="title"
        class="case_header-title"
      >
        Convenient shuttle <br>
        bus service
      </h1>
      <p
        slot="description"
        class="case_header-description"
      >
        Mad Devs helped GoDee with developing feature-rich software to re-invent <br>
        public mobility by building new smart ways of a daily commute.
      </p>
    </CaseHeader>
    <About />
    <IssuesAndSolution />
    <ParallaxImage />
    <IssuesAndSolutionCards />
    <section class="container_full case_video-wrapper m-60_top media-m-24_top m-96_bottom media-m-48_bottom">
      <iframe
        v-lazy-load
        width="1440"
        height="809"
        data-src="https://www.youtube.com/embed/_XCRbXmTUwI"
        frameborder="0"
        allowfullscreen
      />
    </section>
    <Collaboration />
    <Mvp />
    <BeforeAfter />
    <DevelopmentGoals />
    <PreviewCards />
    <IntellegentManagementPanel :is-iphone="isIphone" />
    <TripsMonitor :is-iphone="isIphone" />
    <RouteOptimisation :is-iphone="isIphone" />
    <RealTimeETA />
    <InfrastructureScheme />
    <TechnologyStack />
    <Monitoring />
    <StableScalableSolution />
    <Team />
    <CaseFooter
      link="/case-studies/namba-food/"
      class-name="namba-food"
    >
      <img
        slot="icon"
        v-lazy-load
        :data-src="require(`@/assets/img/Studies/svg/nambafood-footer.svg`)"
        width="200"
        height="63"
        alt="Namba Food"
        class="case_logotype-namba-food"
      >
      Namba Food <br>
      Top Delivery Service in <br class="case_mobile-break-namba-food">
      Central Asia
    </CaseFooter>
  </main>
</template>

<script>
import CaseHeader from '@/components/Cases/shared/CaseHeader'
import About from '@/components/Cases/godee/About'
import IssuesAndSolution from '@/components/Cases/godee/IssuesAndSolution'
import ParallaxImage from '@/components/Cases/godee/ParallaxImage'
import IssuesAndSolutionCards from '@/components/Cases/godee/IssuesAndSolutionCards'
import Collaboration from '@/components/Cases/godee/Collaboration'
import Mvp from '@/components/Cases/godee/Mvp'
import BeforeAfter from '@/components/Cases/godee/BeforeAfter'
import DevelopmentGoals from '@/components/Cases/godee/DevelopmentGoals'
import PreviewCards from '@/components/Cases/godee/PreviewCards'
import IntellegentManagementPanel from '@/components/Cases/godee/IntellegentManagementPanel'
import TripsMonitor from '@/components/Cases/godee/TripsMonitor'
import RouteOptimisation from '@/components/Cases/godee/RouteOptimisation'
import RealTimeETA from '@/components/Cases/godee/RealTimeETA'
import InfrastructureScheme from '@/components/Cases/godee/InfrastructureScheme'
import TechnologyStack from '@/components/Cases/godee/TechnologyStack'
import Monitoring from '@/components/Cases/godee/Monitoring'
import StableScalableSolution from '@/components/Cases/godee/StableScalableSolution'
import Team from '@/components/Cases/godee/Team'
import CaseFooter from '@/components/Cases/shared/CaseFooter'
import isIphoneMixin from '@/mixins/isIphoneMixin'
import IntersectionObserverMixin from '@/mixins/IntersectionObserverMixin'
import playVideo from '@/helpers/playVideo'

const observerOptions = {
  root: null,
  rootMargin: '0px 0px -2px 0px',
  threshold: 0.8,
}

export default {
  name: 'Main',
  components: {
    CaseHeader,
    About,
    IssuesAndSolution,
    ParallaxImage,
    IssuesAndSolutionCards,
    Collaboration,
    Mvp,
    BeforeAfter,
    DevelopmentGoals,
    PreviewCards,
    IntellegentManagementPanel,
    TripsMonitor,
    RouteOptimisation,
    RealTimeETA,
    InfrastructureScheme,
    TechnologyStack,
    Monitoring,
    StableScalableSolution,
    Team,
    CaseFooter,
  },

  mixins: [
    isIphoneMixin,
    IntersectionObserverMixin(
      ['trip-request-map', 'trip-monitor', 'route-optimization'],
      playVideo,
      observerOptions,
    ),
  ],

  data() {
    return {
      logo: {
        width: 293,
        height: 130,
        folder: 'godee',
        file: 'godee-logo',
        alt: 'Godee Logo',
      },
    }
  },
}
</script>
