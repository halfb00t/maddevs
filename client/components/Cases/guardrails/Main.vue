<template>
  <section>
    <About />
    <StoryBehindProject />
    <ProjectIssues />
    <Dashboard />
    <ApiForDashboard />
    <CliandKubernetes />
    <TechnicalAgility />
    <AgilityIsKey />
  </section>
</template>

<script>
import About from '@/components/Cases/guardrails/About'
import StoryBehindProject from '@/components/Cases/guardrails/StoryBehindProject'
import ProjectIssues from '@/components/Cases/guardrails/ProjectIssues'
import Dashboard from '@/components/Cases/guardrails/Dashboard'
import ApiForDashboard from '@/components/Cases/guardrails/ApiForDashboard'
import CliandKubernetes from '@/components/Cases/guardrails/CliandKubernetes'
import TechnicalAgility from '@/components/Cases/guardrails/TechnicalAgility'
import AgilityIsKey from '@/components/Cases/guardrails/AgilityIsKey'

export default {
  name: 'Main',
  components: {
    About,
    StoryBehindProject,
    ProjectIssues,
    Dashboard,
    ApiForDashboard,
    CliandKubernetes,
    TechnicalAgility,
    AgilityIsKey,
  },
}
</script>
