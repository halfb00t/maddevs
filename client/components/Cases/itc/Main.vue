<template>
  <section>
    <div id="case-first-section" />
    <LazyHydrate when-visible>
      <About />
    </LazyHydrate>
    <LazyHydrate when-visible>
      <ProjectMission />
    </LazyHydrate>
    <LazyHydrate when-visible>
      <Challenges />
    </LazyHydrate>
    <LazyHydrate when-visible>
      <SavingTime />
    </LazyHydrate>
    <LazyHydrate when-visible>
      <CollectingDataForAnalysis />
    </LazyHydrate>
    <LazyHydrate when-visible>
      <DevelopingTheInnovation />
    </LazyHydrate>
    <LazyHydrate when-visible>
      <Technologies />
    </LazyHydrate>
    <LazyHydrate when-visible>
      <Team />
    </LazyHydrate>
    <LazyHydrate when-visible>
      <FuturePlans />
    </LazyHydrate>
    <LazyHydrate when-visible>
      <Partnership />
    </LazyHydrate>
  </section>
</template>

<script>
import LazyHydrate from 'vue-lazy-hydration'
import IntersectionObserverMixin from '@/mixins/IntersectionObserverMixin'
import countUp from '@/helpers/countUp'

export default {
  name: 'Main',
  components: {
    LazyHydrate,
    About: () => import('@/components/Cases/itc/About'),
    ProjectMission: () => import('@/components/Cases/itc/ProjectMission'),
    Team: () => import('@/components/Cases/itc/Team'),
    Technologies: () => import('@/components/Cases/itc/Technologies'),
    Challenges: () => import('@/components/Cases/itc/Challenges'),
    SavingTime: () => import('@/components/Cases/itc/SavingTime'),
    CollectingDataForAnalysis: () => import('@/components/Cases/itc/CollectingDataForAnalysis'),
    DevelopingTheInnovation: () => import('@/components/Cases/itc/DevelopingTheInnovation'),
    FuturePlans: () => import('@/components/Cases/itc/FuturePlans'),
    Partnership: () => import('@/components/Cases/itc/Partnership'),
  },

  mixins: [
    IntersectionObserverMixin(
      ['itc-aim-1', 'itc-aim-2', 'itc-aim-3', 'reams', 'sheets-of-paper'],
      countUp,
    ),
  ],
}
</script>
