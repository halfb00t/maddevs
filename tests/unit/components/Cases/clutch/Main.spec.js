import { render } from '@testing-library/vue'
import { shallowMount } from '@vue/test-utils'
import Main from '@/components/Cases/clutch/Main'
import '../../../__mocks__/intersectionObserverMock'

const directives = {
  'lazy-load': () => {},
  'mad-parallax': () => {},
}

describe('Main component', () => {
  it('should render correctly', () => {
    const { container } = render(Main, {
      mocks: {
        $getMediaFromS3: () => 'img.jpg',
      },
      stubs: [
        'NuxtLink',
        'TextQuote',
        'Picture',
        'LazyHydrate',
        'About',
        'WhyClutch',
        'TechChallenges',
        'OutStuffing',
        'Microservices',
        'DataAndPasswords',
        'WidgetSpeed',
        'DataProcessed',
        'DevOps',
        'Workflow',
        'Pipelines',
        'Standup',
        'TechStack',
        'Review',
      ],
      directives,
    })
    expect(container).toMatchSnapshot()
  })

  describe('Dynamic imports Clutch component Main', () => {
    it('should correctly import components', async () => {
      const container = shallowMount(Main, {
        mocks: {
          $getMediaFromS3: () => 'img.jpg',
        },
        directives,
      })

      const About = await container.vm.$options.components.About.call()
      const WhyClutch = await container.vm.$options.components.WhyClutch.call()
      const TechChallenges = await container.vm.$options.components.TechChallenges.call()
      const OutStuffing = await container.vm.$options.components.OutStuffing.call()
      const Microservices = await container.vm.$options.components.Microservices.call()
      const DataAndPasswords = await container.vm.$options.components.DataAndPasswords.call()
      const DevOps = await container.vm.$options.components.DevOps.call()
      const Workflow = await container.vm.$options.components.Workflow.call()
      const Pipelines = await container.vm.$options.components.Pipelines.call()
      const Standup = await container.vm.$options.components.Standup.call()
      const TechStack = await container.vm.$options.components.TechStack.call()
      const Review = await container.vm.$options.components.Review.call()
      const WidgetSpeed = await container.vm.$options.components.WidgetSpeed.call()
      const DataProcessed = await container.vm.$options.components.DataProcessed.call()

      expect(About.default.name).toBe('About')
      expect(WhyClutch.default.name).toBe('WhyClutch')
      expect(TechChallenges.default.name).toBe('TechChallenges')
      expect(OutStuffing.default.name).toBe('OutStuffing')
      expect(Microservices.default.name).toBe('Microservices')
      expect(DataAndPasswords.default.name).toBe('DataAndPasswords')
      expect(DevOps.default.name).toBe('DevOps')
      expect(Workflow.default.name).toBe('Workflow')
      expect(Pipelines.default.name).toBe('Pipelines')
      expect(Standup.default.name).toBe('Standup')
      expect(TechStack.default.name).toBe('TechStack')
      expect(Review.default.name).toBe('Review')
      expect(WidgetSpeed.default.name).toBe('WidgetSpeed')
      expect(DataProcessed.default.name).toBe('DataProcessed')
    })
  })
})
