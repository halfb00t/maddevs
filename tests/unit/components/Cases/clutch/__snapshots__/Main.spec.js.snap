// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main component should render correctly 1`] = `
<div>
  <section>
    <lazyhydrate-stub
      idletimeout="2000"
      whenvisible="[object Object]"
    >
      <about-stub />
    </lazyhydrate-stub>
     
    <lazyhydrate-stub
      idletimeout="2000"
      whenvisible="[object Object]"
    >
      <whyclutch-stub />
    </lazyhydrate-stub>
     
    <lazyhydrate-stub
      idletimeout="2000"
      whenvisible="[object Object]"
    >
      <techchallenges-stub />
    </lazyhydrate-stub>
     
    <lazyhydrate-stub
      idletimeout="2000"
      whenvisible="true"
    >
      <outstuffing-stub />
    </lazyhydrate-stub>
     
    <lazyhydrate-stub
      idletimeout="2000"
      whenvisible="true"
    >
      <microservices-stub />
    </lazyhydrate-stub>
     
    <lazyhydrate-stub
      idletimeout="2000"
      whenvisible="true"
    >
      <dataandpasswords-stub />
    </lazyhydrate-stub>
     
    <lazyhydrate-stub
      idletimeout="2000"
      whenvisible="true"
    >
      <widgetspeed-stub />
    </lazyhydrate-stub>
     
    <lazyhydrate-stub
      idletimeout="2000"
      whenvisible="true"
    >
      <dataprocessed-stub />
    </lazyhydrate-stub>
     
    <lazyhydrate-stub
      idletimeout="2000"
      whenvisible="true"
    >
      <devops-stub />
    </lazyhydrate-stub>
     
    <lazyhydrate-stub
      idletimeout="2000"
      whenvisible="true"
    >
      <workflow-stub />
    </lazyhydrate-stub>
     
    <lazyhydrate-stub
      idletimeout="2000"
      whenvisible="true"
    >
      <pipelines-stub />
    </lazyhydrate-stub>
     
    <lazyhydrate-stub
      idletimeout="2000"
      whenvisible="true"
    >
      <standup-stub />
    </lazyhydrate-stub>
     
    <lazyhydrate-stub
      idletimeout="2000"
      whenvisible="true"
    >
      <techstack-stub />
    </lazyhydrate-stub>
     
    <lazyhydrate-stub
      idletimeout="2000"
      whenvisible="true"
    >
      <review-stub />
    </lazyhydrate-stub>
  </section>
</div>
`;
